Issue 316:  fixing bugs (#449)
ubuntu/ubuntu-project-docs: docs/contributors/bug-fix/bugfixing-workflow.rst

After selecting a bug to fix, identify the source package that contains the code related to the issue.

> Start by identifying the name of the binary package. If the program or file is installed on your system, you can identify the package that owns it using:

Reviewer: ```suggestion
Start by identifying the name of the binary package. To identify the package that owns a file installed on your system, use:
```

In the preceding output, the part before the colon is the name of the binary package.

.. note::

>     If the package is not installed on your system, you can use ``apt-file`` to search across all packages. However, ``apt-file`` is not installed by default and must be set up before use:

Reviewer: ```suggestion
    For packages not installed on your system, use ``apt-file`` to search across all packages. However, ``apt-file`` is not installed by default and must be set up before use:
```
